app/application.py

```diff
@@ -547,7 +539,7 @@ def get_id(_id):
                 return res
 
             # Anything else but a token match means we reject the call
-                app.logger.debug("tokens don't match")
+            app.logger.debug("tokens don't match")
```

Comment:
might was well raise this as an info or warning as we might want to trace when a token does not match. this is due to us rejecting a call - so that means someone might be 'miffed off'! hence let's raise this up a little (i think debug will get ignored on live systems)